### PR TITLE
K8s Cronjobs frontend

### PR DESCRIPTION
**Pull request by atran-brex**
## Hey, I just made a Pull Request!

Adding some basic information on Cronjobs / Jobs to the Kubernetes Plugin. Looking for thoughts/feedback on things people would like to see displayed here!

Screen Shot 2021-11-23 at 3 00 32 PM
Screen Shot 2021-11-23 at 3 01 05 PM

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/kubernetes/src/components/JobsAccordions/JobsDrawer.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  return (
+    <KubernetesDrawer
+      object={job}
+      expanded={expanded}
+      kind="Job"
+      renderObject={(jobObj: V1Job) => {
+        return {
+          parallelism: jobObj.spec?.parallelism ?? '???',
+          completions: jobObj.spec?.completions ?? '???',
+          backOffLimit: jobObj.spec?.backoffLimit ?? '???',
```
Comment (freben) [inline plugins/kubernetes/src/components/JobsAccordions/JobsDrawer.tsx:37]: perhaps unfortunate to have another casing for backOff (capital "O") than the official source data has.

File: plugins/kubernetes/src/components/CronJobsAccordions/CronJobsAccordions.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+          )}
+        </Grid>
+        <Grid item>
+          <Typography variant="body1">
+            Schedule:{' '}
+            {cronJob.spec?.schedule
+              ? `${cronJob.spec.schedule} (${cronstrue.toString(
+                  cronJob.spec.schedule,
+                )})`
+              : '???'}
```
Comment (freben) [inline plugins/kubernetes/src/components/CronJobsAccordions/CronJobsAccordions.tsx:85]: maybe more like "N/A" or so instead? :)

File: .changeset/weak-rivers-perform.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes': minor
```
Comment (freben) [inline .changeset/weak-rivers-perform.md:2]: This is not a breaking change right? Then this can be `patch` (since we are on 0.x versions)

**Review by freben — COMMENTED** [review]
Thanks! Note the build errors as well.

File: plugins/kubernetes/src/components/JobsAccordions/JobsDrawer.tsx (hunk 1 above)
Comment (atran-brex) [reply plugins/kubernetes/src/components/JobsAccordions/JobsDrawer.tsx:37]: good catch!

**Comment by jrusso1020** [comment]
@freben mind taking another look when you get a chance 🙏

**Review by Rugvip — APPROVED** [review]
Looks good, thank you! 👍 

Let's :shipit: 😁

**Pull request merged**